### PR TITLE
Fix the inconsistency of errors thrown

### PR DESCRIPTION
**Pull request by qeq66**

### CODE REVIEW

**Comment by qeq66** [comment]
@oxisto

File: map_claims.go [hunk 1]
```diff
@@ -126,20 +126,17 @@ func (m MapClaims) Valid() error {
 	now := TimeFunc().Unix()
```
Comment (oxisto) [inline map_claims.go:126]: You need to remove the unused import to `errors` as well. You do not need to open/close a new PR, you can add new commits to your existing branch.

Comment (qeq66) [reply map_claims.go:125]: All right, handled

**Review by oxisto — APPROVED** [review]
Because this slightly changes the error descriptions (lower vs. uppercase) I propose merging this into the `v5` branch.

**Pull request closed without merging**